weave

```diff
@@ -1345,6 +1300,10 @@ case "$COMMAND" in
         PEER=$1
         call_weave DELETE /peer/$PEER
         ;;
+    launch-dns|stop-dns)
+        echo "$COMMAND is deprecated." >&2
+        exit 0
+        ;;
```

Comment:
Let's be smarter and more helpful here:

- for `launch-dns` echo "The 'launch-dns' command has been removed; DNS is launched as part of 'launch' and 'launch-router'.". And if weave is not running then exit 1.
- for `stop-dns` echo "The 'stop-dns command has been removed; DNS is stopped as part of 'stop' and 'stop-router'.". Always exit 0.